src/fontra/backends/designspace.py

```diff
@@ -1082,6 +1083,26 @@ def packAxisLabels(valueLabels):
     ]
 
 
+def getPostscriptBlueValues(fontInfo):
+    blueValues = getattr(fontInfo, "postscriptBlueValues", [])
+    otherBluesValue = getattr(fontInfo, "postscriptOtherBlues", [])
+    values = blueValues + otherBluesValue
+    return sorted(values)
+
+
+def getZone(value, blueValues):
```

Comment:
Maybe add
```python
    if len(blueValues) % 2:
        blueValues = blueValues[:-1]  # ensure the list has an even number of items
```
Just to be ultra sure. (Although UFO requires the list to be of even length)

Reply:
done.